lifecycle-operator/controllers/lifecycle/keptnworkloadversion/controller.go

```diff
@@ -72,7 +74,20 @@ type KeptnWorkloadVersionReconciler struct {
 //
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/reconcile
-//
+func NewReconciler(client client.Client, scheme *runtime.Scheme, iEvent controllercommon.IEvent, log logr.Logger, keptnMeters apicommon.KeptnMeters, spanHandler *telemetry.SpanHandler, tracerFactory telemetry.TracerFactory, iSchedulingGatesHandler controllercommon.ISchedulingGatesHandler, tracer trace.Tracer) *KeptnWorkloadVersionReconciler {
```

Comment:
I don't think we need a separate function for creating the reconciler, let's just  use the struct

Reply:
Fixed. I removed the function.